File: srv/Models/Recipe.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace srv.Models
{
    public class Recipe
    {
        public string Id { get; set; }
        public string Name { get; set; }
        public string CategoryId { get; set; }
        public int TimeOfPreparation { get; set; }
        public int LevelOfDifficulty { get; set; }
        public DateTime AddDate { get; set; }
        public List<string> Ingredients { get; set; }
        public List<string> PreparationMethod { get; set; }
        public string UseId { get; set; }
        public string ImgRouting { get; set; }

    }
}

File: src/app/Recipe.ts
export class Recipe{
    constructor (public Id:string,public Name:string,public CategoryId:string,public TimeOfPreparation:number,
        public LevelOfDifficulty:number,public AddDate:Date,public Ingredients:string[],
       public PreparationMethod:string[] ,public UseId:string,public ImgRouting:string
        )
    {
        
    }
 }